src/question-service/src/routes/question-routes.js

```diff
@@ -190,6 +191,6 @@ router.put("/questions/:id", questionController.updateQuestion);
  *  "error": "Question not found"
  * }
  */
-router.delete("/questions/:id", questionController.deleteQuestion);
+router.delete("/questions/:id", auth.authenticate, questionController.deleteQuestion);
```

Comment:
```suggestion
router.delete("/questions/:id", auth.authenticate, auth.checkAuthorization, questionController.deleteQuestion);
```
Could you write a middleware function to check authorization too? We need to restrict Create, Update and Delete operations to admins only